<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSettingDevice</class>
 <widget class="QMainWindow" name="VSettingDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/image/settingdevice.jpg</normaloff>:/image/settingdevice.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>     Thiết bị     </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Cài đặt mã quản lý (S/N) cho thiết bị. </string>
      </property>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SearchDevice">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>151</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tìm thiết bị</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_SerialNumber">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>301</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>1102 123456789</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>461</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Nhập mã (12 kí tự) hoặc nhấn &quot;Tìm thiết bị&quot;.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SetupSN">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cài đặt S/N</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_SN_Increase">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>311</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Tăng S/N lên một đơn vị mỗi khi cài đặt.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>461</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chú ý:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>- Mã quản lý gồm 12 kí tự </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Mã phân nhóm (phân loại thiết bị, luôn cố định)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>461</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>- Nếu có lỗi khi cài đặt thì cần rút cáp kết nối và cắm lại.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_6">
       <property name="geometry">
        <rect>
         <x>342</x>
         <y>240</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>bao gồm:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_7">
       <property name="geometry">
        <rect>
         <x>222</x>
         <y>240</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>1102</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_8">
       <property name="geometry">
        <rect>
         <x>262</x>
         <y>240</y>
         <width>74</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>12345678</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>D - 12345678:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_10">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>260</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>G - 1102 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_Info_11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>280</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Mã định danh thiết bị, thay đổi theo từng thiết bị</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_setG">
       <property name="geometry">
        <rect>
         <x>46</x>
         <y>69</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>G (0)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN_setD">
       <property name="geometry">
        <rect>
         <x>47</x>
         <y>87</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>D (0)</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>     Phần mềm     </string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_SN_Exit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>414</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
